Compile ASTJsonConverter.cpp failed to "error C3861: 'iter_swap': identifier not found" and "error C2672: 'ranges::iter_swap': no matching overloaded function found"

Compile ASTJsonConverter.cpp failed to "error C3861: 'iter_swap': identifier not found" and "error C2672: 'ranges::iter_swap': no matching overloaded function found", can you help look? Thank you very much

**Environment**
- Compiler version: VS2019 (16.11.2)
- Operating system: Windows server 2016

**Reproduce steps:**
1. git clone https://github.com/ethereum/solidity
2. cd solidity
3. mkdir build_amd64
4. cd build_amd64
5. cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DBoost_INCLUDE_DIR="F:\tools\boost_1_67_0\x64" ..
6. msbuild /p:Configuration=Release;Platform=x64 solidity.sln /t:Rebuild /m

**Error log:**
[msbuild_x64.log](https://github.com/ethereum/solidity/files/7231141/msbuild_x64.log)


**Error info:**
1. F:\solidity\build_mad64\deps\include\range/v3/algorithm/sort.hpp(92,25): error C3861: 'iter_swap': identifier not found (compiling source file F:\solidity\libsolidity\ast\ASTJsonConverter.cpp) [F:\solidity\build_mad64\libsolidity\solidity.vcxproj]
2. F:\solidity\build_mad64\deps\include\range/v3/algorithm/sort.hpp(92,25): error C2672: 'ranges::iter_swap': no matching overloaded function found (compiling source file F:\solidity\libsolidity\ast\ASTJsonConverter.cpp) [F:\solidity\build_mad64\libsolidity\solidity.vcxproj]

**Note:**
These errors is due to compile ASTJsonConverter.i (cl /EHsc /permissive- /std:c++20 /TP ASTJsonConverter.i) 
[ASTJsonConverter.i.zip](https://github.com/ethereum/solidity/files/7231159/ASTJsonConverter.i.zip)

--- a/libsolidity/ast/ASTJsonExporter.cpp
+++ b/libsolidity/ast/ASTJsonExporter.cpp
@@ -37,8 +37,6 @@
 
 #include <boost/algorithm/string/join.hpp>
 
-#include <range/v3/algorithm/sort.hpp>
-
 #include <utility>
 #include <vector>
 #include <algorithm>
@@ -629,7 +627,7 @@ bool ASTJsonExporter::visit(InlineAssembly const& _node)
 
 	Json::Value externalReferencesJson = Json::arrayValue;
 
-	ranges::sort(externalReferences);
+	std::sort(externalReferences.begin(), externalReferences.end());
 	for (Json::Value& it: externalReferences | ranges::views::values)
 		externalReferencesJson.append(std::move(it));
 

